src/uwtools/tests/test_cli.py

```diff
@@ -165,6 +150,23 @@ def test__check_template_render_vals_args_noop_explicit_valsfmt():
     assert cli._check_template_render_vals_args(args) == args
 
 
+def test__check_verbosity_fail(capsys):
+    log.setLevel(logging.INFO)
+    args = {STR.quiet: True, STR.verbose: True}
+    with raises(SystemExit):
+        cli._check_verbosity(args)
+    assert (
+        "Specify at most one of %s, %s, or %s"
+        % (cli._switch(STR.debug), cli._switch(STR.quiet), cli._switch(STR.verbose))
+        in capsys.readouterr().err
+    )
```

Comment:
Sorry if I'm missing it, but shouldn't we also test that `--quiet` and `--debug` fails? Maybe that could be done with e.g.
```
@pytest.mark.parametrize("flag", (STR.debug, STR.verbose))
def test__check_verbosity_fail(capsys, flag):
    log.setLevel(logging.INFO)
    args = {STR.quiet: True, flag: True}
    ...
```

I think that would expand the test in to one each for `--debug` and `--verbose`.